site/blog/2022/v2.10.1.md

```diff
@@ -0,0 +1,34 @@
+---
+title: v2.10.1
+date: 2022-09-14
+tags:
+    - releases
+---
+
+## VisiData v2.10.1 Released
```

Comment:
The title is usually something like:

```
## [v2.10.1](https://github.com/saulpw/visidata/releases/tag/v2.10.1) (2021-09-14)
```

Where the date is the date of the release. That link will not work for now, but will work once we officially release and create the tag on VisiData.